acousticbrainz/views/data.py

```diff
@@ -44,21 +46,21 @@ def view_high_level(mbid):
 
 @data_bp.route("/<uuid:mbid>", methods=["GET"])
 def summary(mbid):
-    lowlevel, highlevel, genres, moods, other = get_summary_data(mbid)
-
-    info = _get_track_info(mbid, lowlevel['metadata'])
-
+    summary = get_summary_data(mbid)
+    
+    info = _get_track_info(mbid, summary['lowlevel']['metadata'] if summary['lowlevel'] else None)
```

Comment:
if you throw an exception from `get_summary_data` you can clean up this call:

```python
try:
    summary = get_summary_data(mbid)
    metadata = summary['lowlevel'].get('metadata') # you get `None` for free with .get
    template = ""
    args = {"some": "args_for_template"}
except TheException:
    metadata = None
    template = ""
    args = {"some": "args_for_template"}
info = _get_track_info(mbid, metadata)
tomahawk_url = _get_tomahawk_url(info) # since we have musicbrainz data we may as well use that
...
return render_template(template, **args)
```